src/public/publicAPIs.ts

```diff
@@ -352,7 +352,7 @@ export function shareDeepLink(deepLinkParameters: DeepLinkParameters): void {
  * @param deepLink deep link.
  */
 export function executeDeepLink(deepLink: string, onComplete?: (status: boolean, reason?: string) => void): void {
-  ensureInitialized(FrameContexts.content, FrameContexts.sidePanel, FrameContexts.task);
+  ensureInitialized(FrameContexts.content, FrameContexts.sidePanel, FrameContexts.settings, FrameContexts.task);
```

Comment:
what will happen if deepLink change the route ? will it close setting dialog ?

Reply:
Yes, I'm making the necessary change on the client to close the config dialog when executeDeepLink is called.